v03_pipeline/lib/tasks/delete_project_family_tables.py

```diff
@@ -21,21 +23,24 @@ def complete(self) -> bool:
         )
 
     def run(self):
-        project_table_task: luigi.Target = yield HailTableTask(
-            project_table_path(
+        for sample_type in SampleType:
```

Comment:
same as above, I think ideally we would want to explicitly set which sample type(s) we are deleting